client/libs/ga-scroll-depth.js

```diff
@@ -1,29 +1,33 @@
 // Simplified/rewritten from https://github.com/leighmcculloch/gascrolldepth.js
-
 import { onWindowScrollDebounce$ } from '../js/utils/dom-events';
+import getFlags from '../js/components/launchdarkly';
```

Comment:
We could just call this `featureflags` to abstract out the lib implementation underneath.